app/jobs/regular/group_smtp_email.rb

```diff
@@ -4,13 +4,48 @@
 
 module Jobs
   class GroupSmtpEmail < ::Jobs::Base
+    include Skippable
+
     sidekiq_options queue: 'critical'
 
     def execute(args)
+      return if quit_email_early?
+
       group = Group.find_by(id: args[:group_id])
```

Comment:
Curious, what if the group is deleted? :)

Reply:
![blown](https://user-images.githubusercontent.com/920448/124410752-78665900-dd8e-11eb-9443-37c904a2b070.gif)

Reply:
I think if the group is deleted we can just return? No point logging it haha

Reply:
Yeah will just return in that case I think that's fine. Rare for a group to be deleted...and if it is how is the job even enqueued haha?

Reply:
Super race condition. Group is deleted before the sidekiq job has executed.